index.html

```diff
@@ -0,0 +1,243 @@
+                    <div class="buttons">
+                        <div class="but-html">HTML</div>
+                        <div class="but-boot">Bootstrap</div>
+                        <div class="but-ruby">Ruby</div>
+                    </div>
+                    <div class="orange-button1">
+                        <div class="see-project1">See project</div>
+                    </div>
+                </div>           
+                
+                <div class="grid-item">
+                    <div class="profesional"><h2 class="prof-art">Profesional Art Printing Data</h2></div>
+                    <div class="daily"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                    <div class="buttons">
+                        <div class="but-html">HTML</div>
+                        <div class="but-boot">Bootstrap</div>
+                        <div class="but-ruby">Ruby</div>
+                    </div>
+                    <div class="orange-button1">
+                        <div class="see-project1">See project</div>
+                    </div>
+                </div>    
+                                
+                <div class="grid-item">
+                    <div class="profesional"><h2 class="prof-art">Profesional Art Printing Data</h2></div>
+                    <div class="daily"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                    <div class="buttons">
+                        <div class="but-html">HTML</div>
+                        <div class="but-boot">Bootstrap</div>
+                        <div class="but-ruby">Ruby</div>
+                    </div>
+                    <div class="orange-button1">
+                        <div class="see-project1">See project</div>
+                    </div>
+                </div>
+
+                <div class="grid-item">
+                    <div class="profesional"><h2 class="prof-art">Profesional Art Printing Datas</h2></div>
+                    <div class="daily"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                    <div class="buttons">
+                        <div class="but-html">HTML</div>
+                        <div class="but-boot">Bootstrap</div>
+                        <div class="but-ruby">Ruby</div>
+                    </div>
+                    <div class="orange-button1">
+                        <div class="see-project1">See project</div>
+                    </div>
+                </div>            
+            </div>
+        
+        
+    </section>
+    
+    <section id="skillset">
+        <div class="about">
+                <h2>About me</h2>
```

Comment:
- [ ] Kindly use **h4** instead of **h2** because it optimizes the heading of your portfolio website and enhances the user experience. The `hx` tag is specifically designed for main headings or sub-headings, providing a clear and prominent representation of the page's primary topic or purpose. By utilizing the *h4* tag, you create a visual hierarchy that helps users quickly identify and determine the relevance of the content to their needs and interests. This improves the overall usability and navigation of your website, ensuring a more engaging and effective experience for visitors. Whereas you already have used the `h2` tag on **line no, 33** and the `h3` tag as I mentioned in my above *inline comment* so that's why please maintain a good hierarchy in your development for better accessibility and SEO.

- **Kindly for more information follow [this](https://seosherpa.com/header-tags/) article.**